### PR TITLE
added basic header markup

### PR DESCRIPTION
**Pull request by VladislavaT**

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -1,11 +1,69 @@
-import React from 'react';
+import React, { FC } from 'react';
```
Comment (roshirin) [inline src/components/Header/Header.tsx:1]: I've removed this rule about obligatory React import, so you can just import FC

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -1,11 +1,69 @@
...
+import './header.scss';
+import NiceGadgetsLogo from '../../img/Nice Gadgets.png';
+import Favourite from '../../img/icons/favourite.svg';
+import Basket from '../../img/icons/shopping_bag.svg';
 import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
```
Comment (roshirin) [inline src/components/Header/Header.tsx:7]: Let's make a rearrangement: important react imports first, then classnames, images etc. Just to keep some order :)

File: src/components/Header/Header.tsx [hunk 3]
```diff
@@ -1,11 +1,69 @@
...
+        <ul className="nav__list">
+          <li className="nav__item">
+            <NavLink
+              to="/"
+              className={({ isActive }) =>
+                cn('nav__link', { 'is-active': isActive })
+              }
+            >
+              Home
+            </NavLink>
```
Comment (roshirin) [inline src/components/Header/Header.tsx:28]: It's better to make a separate component for navlink, `<MenuLink />` for example, and use in throughout the navigation.

File: src/components/Header/Header.tsx [hunk 4]
```diff
@@ -1,11 +1,69 @@
...
+        </ul>
+      </nav>
+      <div className="header__actions">
+        <a href="#favorites" className="action">
+          <img
+            src={Favourite}
+            alt="favourite"
+            className="action__image"
+          />
+        </a>
```
Comment (roshirin) [inline src/components/Header/Header.tsx:57]: Basket and favorites must be `<Link>` components.

**Review by roshirin — CHANGES_REQUESTED** [review]
Good start. 👍 

Let's make some improvements.

File: src/components/Header/Header.tsx (hunk 3 above)
Comment (VladislavaT) [reply src/components/Header/Header.tsx:28]: Yep, I also thought of that 
I`ll add it on next commit

File: src/components/Header/Header.tsx [hunk 5]
```diff
@@ -1,11 +1,36 @@
...
+
+  return (
+    <>
+      <header className="header">
+        <div className="header__nav">
+          <Link className="header__logo" to={'/'}>
+            <img src={NiceGadgetsLogo} alt="logo" />
+          </Link>
+
+          <MenuLink />
```
Comment (roshirin) [inline src/components/Header/Header.tsx:25]: MenuLinks

File: src/components/MenuLink/MenuLink.scss
Comment (roshirin) [inline src/components/MenuLink/MenuLink.scss:1]: MenuLinks

File: src/components/MenuLink/MenuLink.tsx
Comment (roshirin) [inline src/components/MenuLink/MenuLink.tsx:1]: MenuLinks

File: src/components/MenuLink/MenuLink.tsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+export const MenuLink = () => (
+  <nav>
+    <ul className="nav__list">
+      {MenuLinks.map(({ to, text }) => (
+        <MenuNavLink key={to} to={to} text={text} />
+      ))}
+    </ul>
+  </nav>
+);
```
Comment (roshirin) [inline src/components/MenuLink/MenuLink.tsx:20]: I think it's better not to map an array here. All these NavLinks have different 'to' and 'text', maybe we'll just code them here, without MenuNavLink component?

**Review by roshirin — CHANGES_REQUESTED** [review]

File: src/components/Header/Header.tsx [hunk 7]
```diff
@@ -1,11 +1,36 @@
...
+            <img src={NiceGadgetsLogo} alt="logo" />
+          </Link>
+
+          <MenuLinks />
+        </div>
+
+        <ActionsLink />
+        <Link to="#burger" className="burger__menu" onClick={handleMenuClick}>
+          <img src={BurgerMenu} alt="burger_menu" />
+        </Link>
```
Comment (roshirin) [inline src/components/Header/Header.tsx:31]: If we don't need router functionality here, only onClick handler, it's better to use `<button>` tag

**Review by roshirin — APPROVED** [review]

**Pull request closed without merging**